package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "landing",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
```

каб захаваць гульню Шыбеніца, трэба каманду білда замяніць на гэтую:
```suggestion
    "build": "vite build && cp -r hangman dist/",
```

---

каманду змяніла на дадзеную